### PR TITLE
added a getter for the mMenuSize property

### PR DESCRIPTION
**Pull request by josomers**
For some very important functionality in my application I added a getter (as there is also a setter), getting the 'menu size'. I thought this could be useful adding to your lovely android-menudrawer library.

### CODE REVIEW

File: library/src/net/simonvt/menudrawer/MenuDrawer.java [hunk 1]
```diff
@@ -957,6 +957,10 @@ void saveState(Bundle state) {
     public void restoreState(Parcelable in) {
         mState = (Bundle) in;
     }
+    
+    public int getMenuSize() {
+    	return mMenuSize;
```
Comment (JakeWharton) [inline library/src/net/simonvt/menudrawer/MenuDrawer.java]: Needs to be indented with four spaces.

**Comment by SimonVT** [comment]
Needs to be moved up to the setter.

**Comment by SimonVT** [comment]
Sorry, didn't see you made the changes. I just added a setter on dev. Thanks though!

**Comment by josomers** [comment]
No problem. I fixed the indentation and the position of the getter in my latest commit. I checked your code on dev and it's exactly what I added in my commit, but I guess you added the getter and not a setter (typo in your comment) ?

**Comment by SimonVT** [comment]
There's both a getter and a setter on dev

**Comment by josomers** [comment]
Yes, I know, but the setter already existed before. Thank you for adding this part into your code.

**Pull request closed without merging**